src/GraphQL/Types/Scalars/StringGraphType.cs

```diff
@@ -2,10 +2,15 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// The String scalar graph type represents a string value.
```

Comment:
```suggestion
    /// The String scalar graph type represents a string value. It is one of the five built-in scalars. 
```

Reply:
Also see summary for IntGraphType:
>  https://graphql.github.io/graphql-spec/June2018/#sec-Int

It is worth linking to the spec section for each of the built-in scalars.